Examples/ReloadDataExample/ReloadDataViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ReloadDataViewController.swift
+//  CollectionKit
+//
+//  Created by yansong li on 2017-09-04.
+//  Copyright © 2017 lkzhao. All rights reserved.
+//
+
+import UIKit
+import CollectionKit
+
+let maxDataCount: Int = 1000
```

Comment:
lets remove this as well. not necessary for the example. Lets just allow user to add infinite amount of cells